src/components/curriculum-section/cycles-list/CyclesSection.js

```diff
@@ -0,0 +1,75 @@
+import { useState } from 'react';
+import useGet from '@hooks/useGet';
+import { endpoints } from '@utils/endpoints';
+import { Skeleton } from 'primereact/skeleton';
+import { Steps } from 'primereact/steps';
+import { Button } from 'primereact/button';
+import Link from 'next/link';
+import style from './CyclesSection.module.scss';
+
+const CyclesSection = ({ curriculumId }) => {
+  const {
+    data: cycles,
+    isLoading: isLoadingCurriculum,
+    isError: isErrorCurriculum,
+  } = useGet(endpoints('cyclesOfCurriculum', curriculumId));
```

Comment:
this hook should be on the father component and passed as props to encapsulate all the db's queries in the same layer of abstraction

Reply:
thanks Ale! I was able to make a bigger redactor thanks to this